File: joseffe-barroso/Python/aula04/for/exercise43.py
sequence = int(input("Qual é o valor do termo?"))

while (sequence <= 0) or (sequence > 50):
    sequence = int(input("O valor da sequência de termos deve ser positivo e menor que cinquenta. Informe novamente: "))

numerator = 1
denominator = 2
sumNumbers = 0

for index in range(1, sequence, 1):
    numerator = (index ** 2) + 1
    denominator = (index ** 3)
    divisao = numerator / denominator
    if (index == 1): print(f'{numerator}')
    else: print(f"{numerator}/{denominator}")
    sumNumbers = sumNumbers + divisao

print('A soma é: %.2f' %sumNumbers)

File: joseffe-barroso/Java/src/entrada_processamento_saida/aula04/For/Exercise43.java
package For;

import java.util.Scanner;

public class Exercise43 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		double n, i, soma, dividendo, divisor, divisao;

		System.out.println("Digite o termo que quer encontrar (valor deve ser positivo e menor que 50): ");
		n = scanner.nextInt();

		while ((n <= 0) || (n > 50)) {
			System.out.println("O valor de N deve ser positivo e menor que 50.");
			System.out.println("Digite o termo que quer encontrar (valor deve ser positivo e menor que 50): ");
			n = scanner.nextInt();
		}

		soma = 0;

		for (i = 1; i <= n; i++) {
			dividendo = (Math.pow(i, 2)) + 1;
			divisor = (Math.pow(i, 3));
			divisao = dividendo / divisor;
			System.out.println(i + ". " + (int) dividendo + "/" + (int) divisor);
			soma = soma + divisao;
			scanner.close();
		}

		System.out.printf("A soma é: %.2f", soma);

	}

}
